specification/cognitiveservices/data-plane/Speech/SpeechToText/stable/v2.0/examples/create_accuracy_test.json

```diff
@@ -20,25 +20,10 @@
     "Content-Type": "application/json"
   },
   "responses": {
-    "401": {
-      "headers": {},
-      "body": {
-        "code": "Unauthorized",
-        "message": "Authentication is required to access the resource."
-      }
```

Comment:
If I am not mistaken, error response should be 
```
{
  "error": {
    "code":
    "message":
  }
}
```

But give it is GA we cannot change it.